MathLib/LinAlg/PETSc/PETScLinearSolver.h

```diff
@@ -34,23 +35,24 @@ namespace MathLib
 /*!
      A class of linear solver based on PETSc rountines.
 
+     TODO does that still hold?
```

Comment:
Addendum: All unrecognized (by ogs) options are passed to the linear solver. The linear solver options in the project file do have precedence because they are parsed later than the command line options.